worker/backend/libcontainerd/client.go

```diff
@@ -51,6 +51,8 @@ type Client interface {
 	) (
 		containers []containerd.Container, err error,
 	)
+
+	Destroy(ctx context.Context, handle string) error
```

Comment:
we should add a comment that describes what's this used for

Reply:
done!